tahoma_description/robots/pod.xacro

```diff
@@ -1,92 +1,118 @@
 <?xml version="1.0"?>
-<robot xmlns:xacro="http://www.ros.org/wiki/xacro" name="pod_recipe">
-  <xacro:macro name="pod_recipe" params="id:=1A">
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro" name="pod">
+  <xacro:arg name="pod_id" default="1A"/>
+  <xacro:arg name="include_flaps" default="true"/>
+  <xacro:arg name="call_macro" default="false"/>
```

Comment:
By convention, xacro files in the `robot` directory of a package should produce a valid URDF when xacro is invoked on them. Macro definitions are expected to live under `/urdf` (strange convention, unless you think of them as the source code for the URDF, and the `robot/` xacro as the (almost) executable form. So there shouldn't need to be a call_macro arg, it should be the only behavior.